org.adempiere.base/src/org/compiere/print/DataEngine.java

```diff
@@ -723,7 +725,15 @@ else if (index == -1)
 				finalSQL.append(by);
 			}
 		}	//	order by
-		
+		else if (format.getAD_ReportView_ID() > 0)
+		{
+			X_AD_ReportView reportView = MReportView.get(Env.getCtx(),format.getAD_ReportView_ID());
```

Comment:
I thinks this must be:

MReportView reportView = MReportView.get(Env.getCtx(),format.getAD_ReportView_ID());

And you don't need to import of X_AD_ReportView